curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-117-red-green-and-blue-tiles.md

```diff
@@ -41,4 +41,23 @@ redGreenBlueTilesTwo();
 
 ```js
 // solution required
+var m = 50; var nmax = 4; var nmin = 2;
```

Comment:
You should not use `var` in these solutions - `var` is a legacy feature of Javascritpt. Also, I don't sea a reason for these global variables.
```suggestion
```